publication-commons/src/main/java/no/unit/nva/publication/model/business/PublicationWorkflow.java

```diff
@@ -1,14 +1,16 @@
-package no.unit.nva.publication.events.handlers.tickets.identityservice;
+package no.unit.nva.publication.model.business;
 
 import static java.lang.String.format;
 import static java.util.Arrays.stream;
 import static java.util.stream.Collectors.joining;
 import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.fasterxml.jackson.annotation.JsonValue;
+import nva.commons.core.JacocoGenerated;
 import nva.commons.core.SingletonCollector;
 
 public enum PublicationWorkflow {
-    
+    UNSET("null"),
```

Comment:
Since I had to include it, in all ticket types, they need an unset value, as they don't use it, and avoiding nulls, let all class copying/transformation tests pass.